### PR TITLE
Static page links

### PR DESCRIPTION
**Pull request by tan-90**
Adds links to static pages to the footer.
Breaks up the top navbar into left and right nav.

Depends on feature/static-pages

### CODE REVIEW

File: src/components/Header/index.tsx [hunk 1]
```diff
@@ -75,3 +90,32 @@ export default function Header() {
     </>
   )
 }
+
+interface DropdownListProps {
```
Comment (Lisenish) [inline src/components/Header/index.tsx:94]: Let's move it to separate file it's complex enough component and it has big chunk of the styles

File: src/components/MobileNavbar/index.tsx [hunk 2]
```diff
@@ -91,3 +88,35 @@ export default function MobileNavbar() {
     </>
   )
 }
+
+interface DropdownMenuProps {
```
Comment (Lisenish) [inline src/components/MobileNavbar/index.tsx:92]: Same here

File: src/components/Header/index.tsx [hunk 3]
```diff
@@ -10,6 +10,9 @@ import Container from '../common/Container'
 import { isBrowser } from '../../helpers/detector'
 import Modal from '../../components/common/Modal'
 import ContactSalesForm from '../../components/ContactSalesForm'
+import { useCaseLinks } from '../../constants/content'
+import classNames from 'classnames'
+import { ReactComponent as ExpandMoreSvg } from '../../img/expand-more.svg'
```
Comment (Lisenish) [inline src/components/Header/index.tsx:15]: Do we use it somewhere else? Maybe it should be a local icon?

File: src/constants/content.ts [hunk 4]
```diff
@@ -1 +1,9 @@
 export const baseUrl = 'https://fingerprintjs.com'
+
+export const useCaseLinks = [
+  { title: 'Account Takeover', url: '/account-takeover/', isLocal: true },
```
Comment (Lisenish) [inline src/constants/content.ts:4]: `isLocal` probably excessive since it's true by default?

**Review by Lisenish — CHANGES_REQUESTED** [review]

File: src/components/Navbar/index.tsx [hunk 5]
```diff
@@ -8,28 +7,14 @@ export default function Navbar() {
...
           <a className={nav.link} href='https://dev.fingerprintjs.com'>
-            Documentation
+            Docs
           </a>
           <a className={nav.link} href='mailto:support@fingerprintjs.com'>
             Support
           </a>
           <a className={nav.link} href='https://dashboard.fingerprintjs.com/login' target='_blank' rel='noreferrer'>
-            Log In
+            Login
```
Comment (Lisenish) [inline src/components/Navbar/index.tsx:17]: Is it intentional change? Maybe we should change it everywhere then? It's Log In in other places

File: src/constants/content.ts (hunk 4 above)
Comment (tan-90) [reply src/constants/content.ts:4]: I added this was to make it compatible with the type the footer is expecting:
```
interface FooterLinkSection {
  title: string
  links: { title: string; url: string; isLocal?: boolean }[]
}
```

The footer considers `false` as the default, so changing that would risk breaking something.

Comment (tan-90) [reply src/constants/content.ts:4]: Or I could just remove it and then map each link to the expected type on the Footer. I'm ok with either solution.

File: src/components/Header/index.tsx (hunk 3 above)
Comment (tan-90) [reply src/components/Header/index.tsx:15]: We use it on the Collapsible as well, but it's local for the collapsible. I'll fix that.

File: src/components/Navbar/index.tsx (hunk 5 above)
Comment (tan-90) [reply src/components/Navbar/index.tsx:17]: It was written Login on the list Savannah provided, but I forgot to check other places. I'll confirm with her just to make sure.

Comment (tan-90) [reply src/components/Navbar/index.tsx:17]: Discussed with Savannah, changed all occurrences to "login".

**Review by Lisenish — APPROVED** [review]

**Pull request merged**